Valid Perfect Square.cpp

```diff
@@ -0,0 +1,16 @@
+class Solution {
+    public:
+    bool isPerfectSquare(int num) 
+    {
+        int left = 0 ; 
+        int right = num;
+        
+        while(left<=right)
+        {   long long mid = left + (right-left)/2;
+            if(mid*mid < num) left = mid+1;
+            else if(mid*mid > num) right = mid-1;
+            else return true;
+        }
+        return false;
+    }
+};
```

Comment:
@HarshitaAgarwal09 a cpp code for the same problem already exists..

Reply:
Sorry, didnt see that.

Reply:
No, it doesnt exists. Only python code exists

Reply:
sorry, my Bad.